src/dal/auth.js

```diff
@@ -1,16 +1,11 @@
-const { Pool } = require('pg');
-const pool = new Pool();
-
-pool.on('error', err => {
-    console.error('Unexpected error on idle client', err);
-});
+const pool = require('./dal-pool');
```

Comment:
Cool! I didn't know if you'd like this idea, but I gotta admit the code looks really clean as a side benefit 😄